jenkins/scripts/bml_integration_test.sh

```diff
@@ -19,6 +19,13 @@ CI_DIR="$(dirname "$(readlink -f "${0}")")"
 # shellcheck disable=SC1090
 source "${CI_DIR}/utils.sh"
 
+if [ "${DISTRIBUTION}" == "ubuntu" ]
+then
+  IMAGE_OS="Ubuntu"
+else
+  IMAGE_OS="Centos"
+fi
```

Comment:
I guess we could hard code it to "Centos" for the BML but this keeps things similar to the other pipelines.

Reply:
One day Ubuntu might work in the BML.